### DISCUSSION TITLE
Como eu consigo pegar a quantidade vendida (exata) de produtos de um vendedor

### DISCUSSION THREAD

**Discussion by caioefdds** [rodurma/forum-meli#9] [Geral]
Eu consegui alguns dados pelo 
 `/sites/$SITE_ID/search?seller_id=$SELLER_ID`
 
 Porém no momento de pegar a quantidade vendida, ele utiliza ranges.
 
 
![image](https://user-images.githubusercontent.com/56234204/150972391-397b1dec-db9a-4d4e-83aa-fdebcac406fc.png)

E o fato de realizar isso, estraga as métricas que estou utilizando no meu site.

Quando utilizo a rota: `/orders/search?seller=$SELLER_ID`

Ele aparentemente só permite que possa realizar a consulta, se a aplication criada lá no ML seja do mesmo dono.

Alguém pode me auxiliar?

**Comment by rgabriel15** [comment]
Oi Caio. Conseguiu encontrar um meio de obter a quantidade total (exata) de vendas?

**Comment by herbetdesign** [comment]
Também não estou conseguindo.
